README.md

```diff
@@ -19,6 +47,54 @@ For formatting code automatically please use `poetry run autopep8 --in-place --r
 
 For linting the codebase run `poetry run flake8 --ignore=E501,E402 .`
 
-Updatebot is currently in early, active development with a lot of churn.
+Updatebot is currently in active development with a lot of churn. We welcome patches and bugfixes, but encourage you to reach out to jewilde or tom ritter before spending much time to ensure we aren't addressing your issue simultaneously.
+
+### How it works
+ - Updatebot runs as a Linux-based cron job in mozilla-central every 6 hours (defined in [.cron.yml](https://searchfox.org/mozilla-central/source/.cron.yml)).  (There is a windows cron job in development, but ignore it for now.) This job:
+  1. Runs in the [Updatebot Docker Image](https://searchfox.org/mozilla-central/source/taskcluster/docker/updatebot)
+  - [Searches](https://github.com/mozilla-services/updatebot/blob/c9133c4f2c15b30438fe6721ef7f490472851de4/components/libraryprovider.py#L122-L129) the tree for [moz.yaml files](https://searchfox.org/mozilla-central/search?q=moz.yaml&case=true&path=) that [define an enabled Updatebot task](https://searchfox.org/mozilla-central/rev/83e67336083df9f9a3d1e0c33f2ba19703d57161/media/libdav1d/moz.yaml#40-43)
```

Comment:
Well the commit hash will ensure we don't have to update the link to point to the right place, but it does mean it might be out of date....  I think people typically understand that a code permalink illustrates where the specific code is, and you probably want to look at tip and find the same code.